stress_test_model_eq_cb.R

```diff
@@ -68,10 +67,8 @@ price_data_version <- cfg_st$price_data_version
 calculation_level <- cfg_st$calculation_level
 company_exclusion <- cfg_st$company_exclusion
 
-# set internal data location, should potentially be moved as this is directly in repo
-# data_location <- paste0(working_location,'data_updated_tool/')
-data_location <- file.path(working_location, data_path())
-# if this is moved out of repo, possibly use set_analysis_inputs_path()
+data_location <- file.path(get_st_data_path(), data_path())
```

Comment:
when this pattern is used (many places in this PR).... I guess the assumption has to be that `data_path()` will always return a **relative** path... I don't know `data_path()` enough to know if that's a safe assumption

Reply:
`data_path()` returns `data-raw/...` by default. `data-raw` can theoretically be changed for something else.. `...` can be an arbitrary number of additional levels and the ultimate file name to be retrieved. so in that sense it is supposed to be used as something returning a relative path within a repo/package/... 
so this should be good as long as the target directory retrieved from `get_st_data_path()` points to the top level of a repo/package or something structured like that.
But I admit I am wondering how reasonable it is to even make two functions out of those.

Reply:
#8